roles/etcd/tasks/pre_upgrade.yml

```diff
@@ -33,10 +33,7 @@
 - name: "Pre-upgrade | remove etcd-proxy if it exists"
   command: "docker rm -f {{item}}"
   with_items: "{{etcd_proxy_container.stdout_lines}}"
-
-- name: "Pre-upgrade | remove etcd-proxy if it exists"
-  command: "docker rm -f {{item}}"
-  with_items: "{{etcd_proxy_container.stdout_lines}}"
```

Comment:
This removes a duplicate declaration ( https://github.com/kubernetes-incubator/kargo/blob/master/roles/etcd/tasks/pre_upgrade.yml#L33 and https://github.com/kubernetes-incubator/kargo/blob/master/roles/etcd/tasks/pre_upgrade.yml#L37 ). If calling `docker rm` twice is necessary i can add it back in.